### DISCUSSION TITLE
Track link/user from ad >> marketing site >> ordering site

### DISCUSSION THREAD

**Discussion by rdyar** [plausible/analytics#3237] [Anything Else]
is there a way to associate the ref from an add click all the way thru a marketing site and then into an ecommerce site?

I have a google add and the link includes /?ref=googlead

That link goes to our marketing site and is tracked as googlead for the referrer and I can see what pages and links they clicked up until they click an Order Now link which takes them to our ecommerce site.

I'd like to be able to see them also on the ecom site with the same referrer.

Both sites are using the same plausible property called "global" so they all are together. I also have both as individual properties.

Ultimately I want to see on the ecom site the same googlead referrer but they will go thru the marketing site first.

**Comment by metmarkosaric** [comment]
hi @rdyar! this should work out of the box as long as the ecommerce site and the marketing site are on the same domain name (or the ecommerce site is on a subdomain of the main domain like store.marketingsite.com). if you have completely different domains for the ecommerce site compared to the marketing site, then it won't work because we don't do any cross domain tracking. in that case, it's best to redirect the buyer back to the main domain to some type of a 'thank you' page after a successful purchase. then you can track the 'thank you' page as a pageview goal and it will work all fine with the original referral source getting the attribution for the purchase

**Comment by rdyar** [comment]
The ecom site is not the same domain and I can't have it redirect back for the success page. But maybe I can do some dns changes to have it be a subdomain, didn't know the cross domain tracking was an issue.

Thanks for the info.
